libtsuba/src/FileFrame.cpp

```diff
@@ -212,6 +212,7 @@ katana::FileFrame::PaddedWrite(
       num_padding_bytes, nbytes, byte_boundry);
   if (num_padding_bytes > 0) {
     uint8_t padding[num_padding_bytes];
+    std::fill(padding, padding + num_padding_bytes, 0);
```

Comment:
hows this?

Reply:
I think the `padding.size()` is sufficient.  The `* sizeof(uint8_t)` seems like it is only correct because `sizeof(uin8_t) == 1`.

Reply:
I'm not so sure, I think its the other way around. 
Write wants the number of bytes, not the number of elements:
```
arrow::Status
katana::FileFrame::Write(const void* data, int64_t nbytes) {
```
which would be:
```
vector.size() * sizeof(vector_type)
```

so in this case 

```
aro_sts = Write(padding.data(), padding.size());
```
would only be correct because `sizeof(uin8_t) == 1`

Reply:
Ugh, I've been writing my own `size()` methods that return byte counts.  Sorry!

Reply:
heh, no problem, honestly I wish Write just figured this out for us. I'll have to add that sometime.